www/blockly

```diff
@@ -1 +1 @@
-Subproject commit 8af40c14a99c7e3b0f94a0d2c1b8f228060c8ec2
+Subproject commit f17610d47bfa51fb770b274cdc3a1dd7dc72108d
```

Comment:
Yeah, you're right. Hm. Any idea how to get rid of it at this point?
I really don't understand how git manages sub-repos. It would probably be best to make this git project not track this repo at all.